src/preset-themes/jsonSchema/jsonSchema.js

```diff
@@ -0,0 +1,227 @@
+import { makeLog } from "../../lib/utils";
+
+const log = makeLog("background");
+
+var Ajv = require("ajv");
+var ajv = new Ajv({
+    allErrors: true
+});
+
+var schema = {
+    "properties": {
+        "title": {
+            "type": "string"
+        },
+        "colors": {
+            "type": "object",
+            "properties": {
+                "toolbar": {
+                    "type": "object",
+                    "properties": {
+                        "h": {
+                            "type": "number",
+                            "maximum": 360,
+                            "minimum": 0
+                        },
+                        "s": {
+                            "type": "number",
+                            "maximum": 100,
+                            "minimum": 0
+                        },
+                        "l": {
+                            "type": "number",
+                            "maximum": 100,
+                            "minimum": 0
+                        },
+                        "a": {
+                            "type": "number",
+                            "maximum": 100,
+                            "minimum": 0
+                        }
+                    },
+                    "required": ["h", "s", "l"]
+                },
+                "toolbar_text": {
+                    "type": "object",
+                    "properties": {
+                        "h": {
+                            "type": "number",
+                            "maximum": 360,
+                            "minimum": 0
+                        },
+                        "s": {
+                            "type": "number",
+                            "maximum": 100,
+                            "minimum": 0
+                        },
+                        "l": {
+                            "type": "number",
+                            "maximum": 100,
+                            "minimum": 0
+                        },
+                        "a": {
+                            "type": "number",
+                            "maximum": 100,
+                            "minimum": 0
+                        }
```

Comment:
Only 2 properties support an alpha channel, so we should remove the `a` where it isn't needed (and possibly throw errors when it isn't wanted but is present):

https://github.com/mozilla/Themer/blob/38933c93a6c304a70b095dfd737b3b2c8ce871d6/src/lib/constants.js#L16-L19

Similarly, we should make sure that the `toolbar` and `toolbar_field` properties have the `"a"` property marked as required.

Reply:
Will do it. :-) 
Thanks for reviewing.